velodrome/prober/blackbox.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: blackbox-prober
+  labels:
+    app: blackbox-prober
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: blackbox-prober
+    spec:
+      containers:
+      - name: blackbox-prober
+        args:
+        - --config.file=/etc/prometheus/prober.yaml
+        image: prom/blackbox-exporter:v0.8.1
+        volumeMounts:
+        - name: config
+          mountPath: /etc/prometheus/
+      volumes:
+      - name: config
+        configMap:
+          name: blackbox-prober-config
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: blackbox-prober-config
+data:
+  prober.yaml: |-
+    modules:
+      http_2xx:
+        prober: http
+        timeout: 5s
+        http:
+          valid_status_codes: [] # Defaults to 2xx
+          method: GET
+          headers:
+            Host: prow.k8s.io
```

Comment:
Why is this specific to `prow.k8s.io`?

Reply:
In HTTP/1.1, the "Host" header must be set. Unfortunately, we can't pass this as an argument to /probe, and there's no intention to support that in the future. I don't believe anything of our's rejects on a mismatched host field, so it's really just there as a stub.